### ISSUE TITLE
map that has single place with data doesnt render properly

### ISSUE THREAD

**Issue by jspeis** [d3plus/d3plus-geomap#22]
### Expected Behavior

Map renders, but with only 1 place (colored as either the highest or lowest value in the scale)

### Current Behavior

Map renders black and gets stuck saying "Loading visualization / powered by d3plus"

<img width="812" alt="screen shot 2018-08-02 at 4 15 29 pm" src="https://user-images.githubusercontent.com/87581/43608596-7a5eb84c-966f-11e8-95c6-d82203efb9a2.png">

```
Geomap.js:76 Uncaught TypeError: this$1._colorScaleClass._colorScale is not a function
    at Geomap.fill (Geomap.js:76)
    at eval (configPrep.js:23)
    at Function.styleLogic (Shape.js:208)
    at SVGPathElement.eval (attr.js:30)
    at Selection.__webpack_exports__.a [as each] (each.js:5)
    at Selection.__webpack_exports__.a [as attr] (attr.js:54)
    at Path._applyStyle (Shape.js:212)
    at Selection.__webpack_exports__.a [as call] (call.js:4)
    at Path.render (Path.js:61)
    at Geomap._draw (Geomap.js:364)
```

### Steps to Reproduce (for bugs)

1. https://jsfiddle.net/jonathan_dw/cedntuxj/1/
2. Note error in javascript console

I tried both the uniform and jenks scales.

### Your Environment

* Browser Name: Chrome 67
* Operating System and Version: OS X 10.13.6

**Comment by davelandry** [comment]
FFFFFFFFFUUUU- I thought I fixed this with #12! Taking a look right now.

**Comment by davelandry** [comment]
@jspeis fixed. JSfiddle is caching the old source code, but if you switch to the non-minified version you can see it working: https://jsfiddle.net/cedntuxj/3/
